sql/core/src/test/scala/org/apache/spark/sql/GeneratorFunctionSuite.scala

```diff
@@ -389,6 +389,31 @@ class GeneratorFunctionSuite extends QueryTest with SharedSparkSession {
         Row(0, 1) :: Row(3, 4) :: Row(6, 7) :: Row(null, null) :: Row(null, null) :: Nil)
     }
   }
+
+  test("SPARK-39061: inline should handle null struct") {
```

Comment:
Seems like this fails in Scala 2.13 for some reasons. I filed a JIRA at https://issues.apache.org/jira/browse/SPARK-39524

Reply:
@HyukjinKwon I will take a look.

Reply:
I resolved that Jira (and a few others) as a dup of SPARK-39520 ("ExpressionSetSuite test failure with Scala 2.13") since those failures are fixed by SPARK-39520's proposed fix.

Reply:
Thanks!